### PR TITLE
Neteleven

### PR DESCRIPTION
**Pull request by reinhardklein-neteleven**
Changes proposed in this pull request:

- Added neteleven as "adopter"

### CODE REVIEW

File: content/adopters/adopters.yaml [hunk 1]
```diff
@@ -25,3 +25,6 @@ adopters:
     url: "https://arineo.com/de/customer-experience/177-iot-with-sap"
     logo: "./logos/arineo.svg"  
     content: "Kyma solves our clients’ need for a flexible and reusable SAP Commerce extension mechanism. It allows them to deliver extraordinary customer experience."
+  - websiteUrl: "https://neteleven.de/"
+    logo: "./logos/neteleven.svg"  
+    content: "Kyma is the way to enhance SAP Commerce platforms with cloud native functionality to get the best of both worlds."
```
Comment (kazydek) [inline content/adopters/adopters.yaml:30]:
```suggestion
    content: "Kyma is the way to enhance SAP Commerce platforms with the cloud-native functionality to make the best out of both worlds."
```

**Comment by kazydek** [comment]
@reinhardklein-neteleven, thank you for your contribution!  I reviewed it and added a small suggestion to the description you provided. Could you also sign the CLA so we could accept your contribution? Thanks in advance :)

File: content/adopters/adopters.yaml (hunk 1 above)
Comment (reinhardklein-neteleven) [reply content/adopters/adopters.yaml:30]: thank you very much. CLA is signed.
Do you really think "the" cloud-native functionality? If you think you are the expert in perfect technical english, so fine for me!

Comment (kazydek) [reply content/adopters/adopters.yaml:30]: @reinhardklein-neteleven, if you prefer not to add "the", that's also fine.

**Pull request merged**